tools/runfiles/runfiles.rs

```diff
@@ -43,6 +43,8 @@ const RUNFILES_DIR_ENV_VAR: &str = "RUNFILES_DIR";
 const MANIFEST_FILE_ENV_VAR: &str = "RUNFILES_MANIFEST_FILE";
 const MANIFEST_ONLY_ENV_VAR: &str = "RUNFILES_MANIFEST_ONLY";
 const TEST_SRCDIR_ENV_VAR: &str = "TEST_SRCDIR";
+const EXTERNAL_GENERATED_FILE_REGEX: &str = "bazel-out[/][^/]+/bin/external/([^/]+)/";
```

Comment:
wdyt of avoiding the regex dep here? I think `str.split('/')` should get you what you want?